src/workerd/api/node/tests/net-nodejs-test.js

```diff
@@ -457,8 +457,7 @@ export const testNetConnectImmediateFinish = {
 // something different than the original Node.js test
 export const testNetConnectKeepAlive = {
   async test() {
-    // Test that setKeepAlive call does not throw.
-    throws(() => new net.Socket({ keepAlive: true }));
+    // throws(() => new net.Socket({ keepAlive: true }));
```

Comment:
Could you please add a comment explaining that we're choosing not to throw on keep alive?